src/count-cats.js

```diff
@@ -1,6 +1,10 @@
 const CustomError = require("../extensions/custom-error");
 
-module.exports = function countCats(/* matrix */) {
-  throw new CustomError('Not implemented');
-  // remove line with error and write your code here
-};
+module.exports = function countCats(arrayTest) {
+  let counter = 0;
+  for(let i=0; i<arrayTest.length; i++){
+
+    counter += arrayTest[i].filter(cats => cats === '^^').length;
+  }
+  return counter
+};
```

Comment:
You could use .forEach instead of 'for' loop. This way you don't need 'i'.

arrayTest.forEach(item => {counter += item.filter....

But the best way to use .reduce here. And you don't need even 'counter'. Also you can use es6 spread operator for arrays here.

return arrayTest.reduce((acc, item) => {
return acc + item.filter(cats => cats === '^^').length;
},0]

Reply:
@DmytroDubravin, fixed. Ready for review